### PR TITLE
Update settings page

### LINKED ISSUE #1571: [Phoenix] Revamp settings page UI to be more webapp
- Link to designs [https://app.zeplin.io/project/58d91e7333cc4a43c790d340/screen/5d653a89a649819b9caa18ff](https://app.zeplin.io/project/58d91e7333cc4a43c790d340/screen/5d653a89a649819b9caa18ff)

### PR DESCRIPTION
**Pull request by lise-toggl**
## :star2: What does this PR do?

Updates the Toggl Button settings page to look like the Toggl web app.

image

Dark mode:
image

## :bug: Recommendations for testing

Open the settings page, turn settings on and off.

## :memo: Links to relevant issues or information

Closes #1571

### CODE REVIEW

**Comment by shantanuraj** [comment]
image

The size of the input field relative to the buttons below it (especially in the custom domain url)
looks a bit random

**Comment by shantanuraj** [comment]
Let's ask support to update this https://support.toggl.com/en/articles/2207027-toggl-button-for-chrome
after we publish phoenix-release-2

It is horribly outdated.

File: src/styles/settings.css [hunk 1]
```diff
@@ -18,267 +33,396 @@ body {
...
 header {
-  width: 100%;
+  height: 66px;
   display: flex;
-  justify-content: center;
-  box-shadow: inset 0px -1px 1px -1px #bbb;
-  position: relative;
+  align-items: center;
+  flex: 0 0 auto;
+  background: white;
```
Comment (shantanuraj) [inline src/styles/settings.css]: The dark mode looks amazing bar the header
image

I think we should add a css-var for it as well `--header-bg-color`

File: src/styles/settings.css [hunk 2]
```diff
@@ -18,267 +33,396 @@ body {
...
 */
+
+#origins-container {
+  display: inline;
+}
+
 .origin-list {
-  border: 1px solid #cccaca;
-  list-style: none;
   max-height: 290px;
```
Comment (shantanuraj) [inline src/styles/settings.css]: Can we change this to `height: 290px;` so the layout doesn't jump on filtering list

File: src/html/settings.html [hunk 3]
```diff
@@ -4,346 +4,399 @@
...
+      </div>
+      <div class="tab" data-tab="integrations">
+        <section>
+          <div class="headline">
+            <h3>Integrations</h3>
+          </div>
+          <p>Enable the tools you want to use with Toggl Button. You can enable everything by clicking the "Enable all" button.</p>
+          <div class="permissions-toolbar" data-id="regular">
+            <input type="text" id="permission-filter" size="44" placeholder="Filter integrations"/>
+            <a href="javascript:void(0)" id="filter-clear">&times;</a>
```
Comment (shantanuraj) [inline src/html/settings.html]: This overflows the parent container and causes a layout jump when you type anything in the integration filter box
image

Also the hit-target is so small, maybe we can postpone cleaning this part for https://github.com/toggl/toggl-button/issues/1572

File: src/html/settings.html [hunk 4]
```diff
@@ -4,346 +4,399 @@
...
-                <label for="enable-auto-tagging">Enable automatic tagging</label>
-                <div class="description">If an integration supports tags or labels, automatically add them to the timer and
-                  create them in Toggl. Not available in all integrations.</div>
-              </div>
-
-              <div>
-                <input type="checkbox" id="enable-auto-tagging">
-              </div>
+      <div class="section">
+        <h3>Button Settings</h3>
```
Comment (shantanuraj) [inline src/html/settings.html]: `Button` seems redundant here, maybe just use `Settings`

File: src/html/settings.html [hunk 5]
```diff
@@ -4,346 +4,399 @@
...
-              </div>
-              <div>
-                <input type="checkbox" id="show_post_start_popup">
-              </div>
+            <span>My Account</span>
+          </li>
+        </ul>
+      </div>
+      <div class="section">
+        <h3>Button Help</h3>
```
Comment (shantanuraj) [inline src/html/settings.html]: Same here `Help` should do the job

File: src/html/settings.html (hunk 4 above)
Comment (lise-toggl) [reply src/html/settings.html]: If we remove it I'd like some other way to show clearly that this is a button page, not a webapp page. Should we talk to product about this?

File: src/html/settings.html (hunk 5 above)
Comment (lise-toggl) [reply src/html/settings.html]: Same as https://github.com/toggl/toggl-button/pull/1595/files#r350984711

File: src/html/settings.html (hunk 3 above)
Comment (lise-toggl) [reply src/html/settings.html]: I'll clean this up, didn't realise size would causes these issues :woman_facepalming:

**Comment by lise-toggl** [comment]
Improved dark style a bit and updated the integration filters - does it look ok on your end now?

**Comment by lise-toggl** [comment]
I didn't really like the filled in version of the icon we looked at yesterday, so spent a while making a tomato - what do you think?

image

**Comment by shantanuraj** [comment]
@lise-toggl I've updated the base branch on the PR
can you rebase your commit onto the `phoenix-release-3` branch?

**Comment by lise-toggl** [comment]
Done, only change was to add the pomo focus setting which worked for me :)

File: src/styles/settings.css [hunk 6]
```diff
@@ -1,15 +1,41 @@
+/*@import url(https://fonts.googleapis.com/css?family=Open+Sans:400italic,400,300,600,700&subset=cyrillic,latin,latin-ext);
+*/
```
Comment (shantanuraj) [inline src/styles/settings.css]: We can delet this

**Pull request merged**